clients/drcachesim/tracer/instru_offline.cpp

```diff
@@ -955,8 +955,7 @@ offline_instru_t::identify_elidable_addresses(void *drcontext, instrlist_t *ilis
         // view by expanding the instr in raw2trace (e.g. using
```

Comment:
I assume this is in preparation for a scatter-gather drmemtrace basic counts test? We could add that in this PR too if possible.

Reply:
Okay. I'm fine with submitting just this one for now.